src/question.ts

```diff
@@ -1439,7 +1439,7 @@ export class Question extends SurveyElement
   protected updateIsAnswered(): void {
     const oldVal = this.isAnswered;
     this.setPropertyValue("isAnswered", this.getIsAnswered());
-    if(oldVal !== this.isAnswered) {
+    if (oldVal !== this.isAnswered) {
```

Comment:
Please remove space between "if" and "("

Reply:
Removed. 

The rest of the file has a space after its "if" statements. Which is the preferred format for the library?